packages/core/test/CoreTest.ts

```diff
@@ -0,0 +1,57 @@
+import { assert, expect } from "chai";
+import mocha from "mocha";
+import { AeroGearConfiguration, ServiceConfiguration } from "../src/config";
+import { AgsCore } from "../src/Core";
+import testConfigJson from "./mobile-config.json";
+
+describe("AeroGear Core Tests", () => {
+
+  const aerogearConfig = testConfigJson as AeroGearConfiguration;
+  let parser: AgsCore;
+
+  beforeEach(() => {
+    parser = new AgsCore();
+    parser.init(aerogearConfig);
```

Comment:
if init returns promise, let's use `async/await` so that beforeEach is fully executed.